src/themes/dark-theme/dark-theme-with-move.js

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import deepMerge from '../../utils/deep-merge.js';
+import {DarkTheme} from './dark-theme.js';
+import {
+  typography as moveTypography,
+  fontTokens as moveFontTokens,
+} from '../move-theme/typography.js';
+import getTypography from '../shared/typography.js';
+
+import type {ThemeT} from '../../styles/types.js';
+
+export const DarkThemeMove: ThemeT = deepMerge(
```

Comment:
Any reason this uses deepMerge instead of createTheme?

Reply:
It's identical to the dark theme with exception of typography so no reason to duplicate dark theme build on top of the light theme values that are default in `createTheme`.